cpp/CMakeLists.txt

```diff
@@ -370,6 +370,7 @@ if(ARROW_ENABLE_TIMING_TESTS)
   add_definitions(-DARROW_WITH_TIMING_TESTS)
 endif()
 
+
```

Comment:
Could you revert needless changes like this?
It seems that there are more needless changes. Could you check again?

Reply:
Darn, I ran auto-lint on everything, sorry,  will remove them.